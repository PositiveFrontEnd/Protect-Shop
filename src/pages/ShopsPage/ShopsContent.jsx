import React from 'react';
import Location from './ShopsSvg/location.svg?react'
import PhoneB from './ShopsSvg/phoneB.svg?react'
import Time from './ShopsSvg/time.svg?react'
import PropTypes from "prop-types";

const ShopsContent = ({ city, address, hours }) => {
    return (
        <div className='shops__content'>
            <h3 className='shops__city'>{city}</h3>
            <div className='shops__contacts'>
                <div className='shops__contacts__item'>
                    <Location /><p className='shops__contacts__text'>Address: {address}</p>
                </div>
                <div className='shops__contacts__item'>
                    <PhoneB /><a className='shops__contacts__text' href="tel:+380505000096">Phone: +380(50) 500
                        00 96</a>
                </div>
                <div className='shops__contacts__item'>
                    <Time /><p className='shops__contacts__text'>Schedule: Mon-Sun: {hours}</p>
                </div>
            </div>
        </div>
    );
};

ShopsContent.propTypes = {
    city: PropTypes.string,
    address: PropTypes.string,
    hours: PropTypes.string
}

export default ShopsContent;