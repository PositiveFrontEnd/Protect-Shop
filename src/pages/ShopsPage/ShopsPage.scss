.shops__title {
  padding: 8px 0;
  font-size: 18px;
  font-family: "Noto Sans", sans-serif;
  line-height: 25px;
  text-align: center;

  @media (width >= 768px) {
    padding: 10px 0;
    font-size: 24px;
    line-height: 33px;
  }
}

.shops__nav {
  display: flex;
  flex-wrap: wrap;
  gap: 4px 12px;
  justify-content: center;
  padding: 8px 16px 16px;

  @media (width>=481px) {
    padding: 8px 30px;
  }

  @media (width >= 768px) {
    padding: 40px 80px 20px;
  }
}

.shops__nav a.active {
  color: var(--white);
  background-color: var(--black);
  border: 1px solid var(--black);
}

.shops {
  display: flex;
  flex-direction: column;

  @media (width>=481px) {
    padding: 0 30px;
  }

  @media (width >= 768px) {
    flex-direction: row-reverse;
    height: 417px;
    padding: 0 80px;
  }
}

.shops__map {
  align-self: center;
  width: 100%;
  margin: 0 -26px;

  @media (width >= 768px) {
    display: flex;
    align-items: center;
    justify-content: center;
    height: 417px;
    overflow: hidden;
  }

  img {
    width: 100%;

    @media (width >= 768px) {
      width: 80%;
      max-width: 740px;
      height: 417px;
      object-fit: cover;
    }
  }
}

.shops__img {

  width: 100%;
  height: 317px;

  @media (width >= 768px) {
    width: 80%;
    max-width: 740px;
    height: 417px;
    object-fit: cover;
  }
}

.shops__marker {
  position: absolute;
  top: 20px;
  left: 20px;
  padding: 15px;
  background: var(--white);
  border: var(--fild-type-color) 1px solid;
  border-radius: 2px;
}

.shops__content__block {
  display: flex;
  flex-direction: column;
  gap: 16px;
  padding: 8px 16px;

  @media (width>=481px) {
    padding: 8px 0;
  }

  @media (width >= 768px) {
    flex-direction: column;
    width: 334px;
    height: 417px;
    overflow: scroll;
  }
}

::-webkit-scrollbar {
  width: 4px;
}

::-webkit-scrollbar-thumb {
  background-color: var(--fild-type-color);
  border-radius: 2px;
}

.shops__content {
  padding: 20px 0 8px;

  .shops__city {
    font-weight: 700;
    font-size: 14px;
    line-height: 19px;
    text-transform: uppercase;
  }
}

.shops__contacts {
  display: flex;
  flex-direction: column;
  gap: 4px;
  margin-top: 12px;

  .shops__contacts__item {
    display: flex;
    gap: 4px;
    align-items: center;
    justify-content: start;
  }

  .shops__contacts__text {
    font-size: 12px;
    line-height: 18px;
  }
}
