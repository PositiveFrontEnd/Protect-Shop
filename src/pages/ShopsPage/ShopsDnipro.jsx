import React from 'react';
import ShopsContent from "src/pages/ShopsPage/ShopsContent";
import ShopsMap from "src/pages/ShopsPage/ShopsMap";

const ShopsDnipro = () => {

    const center = { lat: 48.4486, lng: 35.0266 }
    const markers = [
        { lat: 48.425, lng: 35.0216, address: 'Dnipro, blv. Zorianyi, 1a' },
        { lat: 48.4644, lng: 35.0473, address: 'Dnipro, ave. D.Yavornytskogo, 50' },
    ];

    return (
        <div className='shops container'>
            <ShopsMap zoom={12.5} center={center} markers={markers} />
            <div className='shops__content__block'>
                <ShopsContent city='dnipro' hours='11:00 – 20:00'
                    address='Dnipro, blv. Zorianyi, 1a' />
                <ShopsContent city='dnipro' hours='10:00 – 21:00'
                    address='Dnipro, ave. D.Yavornytskogo, 50' />
            </div>
        </div>
    );
};

export default ShopsDnipro;