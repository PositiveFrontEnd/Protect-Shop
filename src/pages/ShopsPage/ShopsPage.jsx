import React from "react";
import "./ShopsPage.scss";
import {NavLink, Outlet} from "react-router-dom";
import "../../components/Button/Button.scss";
import "../../../src/components/Helpers/Base/Base.scss";

const ShopsPage = () => {

    return (
        <>
            <div>
                <h2 className="shops__title">LIST STORE ADDRESS</h2>
            </div>
            <nav className="shops__nav">
                <NavLink
                    className="button-white button__submitFilter-white"
                    to="kharkiv"
                >
                    kharkiv
                </NavLink>
                <NavLink className="button-white button__submitFilter-white" to="kyiv">
                    kyiv
                </NavLink>
                <NavLink className="button-white button__submitFilter-white" to="odesa">
                    odesa
                </NavLink>
                <NavLink className="button-white button__submitFilter-white" to="lviv">
                    lviv
                </NavLink>
                <NavLink
                    className="button-white button__submitFilter-white"
                    to="dnipro"
                >
                    dnipro
                </NavLink>
            </nav>
            <Outlet/>
        </>
    );
};

export default ShopsPage;
