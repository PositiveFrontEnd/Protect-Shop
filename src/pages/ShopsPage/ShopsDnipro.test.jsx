import React from "react";
import { render } from "@testing-library/react";
import ShopsPage from "./ShopsDnipro";
import { BrowserRouter } from "react-router-dom";
describe("Тестування компонента ShopsPage", () => {
    test("Знімок", () => {
        const shopsPage = render(
            <BrowserRouter>
                <ShopsPage />
            </BrowserRouter>
        );
        expect(shopsPage).toMatchSnapshot();
    });
});
