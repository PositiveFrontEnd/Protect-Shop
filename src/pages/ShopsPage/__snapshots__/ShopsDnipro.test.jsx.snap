// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Тестування компонента ShopsPage > Знімок 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="shops container"
      >
        <div
          class="shops__map"
        />
        <div
          class="shops__content__block"
        >
          <div
            class="shops__content"
          >
            <h3
              class="shops__city"
            >
              dnipro
            </h3>
            <div
              class="shops__contacts"
            >
              <div
                class="shops__contacts__item"
              >
                <svg
                  fill="none"
                  height="12"
                  viewBox="0 0 12 12"
                  width="12"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M5.99922 12C7.05876 12 10.7992 7.244 10.7992 4.66667C10.7992 2.08934 8.65019 0 5.99922 0C3.34825 0 1.19922 2.08934 1.19922 4.66667C1.19922 7.244 4.93968 12 5.99922 12ZM5.99922 6.66667C7.13535 6.66667 8.05635 5.77124 8.05635 4.66667C8.05635 3.5621 7.13535 2.66667 5.99922 2.66667C4.86309 2.66667 3.94209 3.5621 3.94209 4.66667C3.94209 5.77124 4.86309 6.66667 5.99922 6.66667Z"
                    fill="#191919"
                    fill-rule="evenodd"
                  />
                </svg>
                <p
                  class="shops__contacts__text"
                >
                  Address: 
                  Dnipro, blv. Zorianyi, 1a
                </p>
              </div>
              <div
                class="shops__contacts__item"
              >
                <svg
                  fill="none"
                  height="17"
                  viewBox="0 0 16 17"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M1.33791 3.52769C1.27065 2.57483 2.07752 1.83325 3.03079 1.83325H5.07135C5.56069 1.83325 6.00203 2.12809 6.19023 2.58072L7.04351 4.63289C7.25138 5.1328 7.10385 5.71021 6.68185 6.04853L6.14994 6.47495C5.91257 6.66524 5.83187 6.99338 5.97261 7.26307C6.69591 8.64934 7.76491 9.82581 9.06491 10.6777C9.33931 10.8575 9.70178 10.786 9.90651 10.5295L10.4608 9.83527C10.7984 9.41241 11.3746 9.26454 11.8734 9.47287L13.9214 10.3279C14.3731 10.5165 14.6673 10.9588 14.6673 11.4491V12.1577H14.6399V13.4665C14.6399 14.4217 13.8999 15.2303 12.949 15.1626C6.86871 14.7301 1.76896 9.63441 1.33791 3.52769Z"
                    fill="#191919"
                  />
                </svg>
                <a
                  class="shops__contacts__text"
                  href="tel:+380505000096"
                >
                  Phone: +380(50) 500 00 96
                </a>
              </div>
              <div
                class="shops__contacts__item"
              >
                <svg
                  fill="none"
                  height="17"
                  viewBox="0 0 16 17"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_511_22270)"
                  >
                    <path
                      d="M13.9045 6.25251C14.1618 6.25251 14.4036 6.15238 14.5855 5.97049C14.7674 5.78856 14.8676 5.54677 14.8676 5.28952C14.8676 5.03227 14.7674 4.79048 14.5855 4.60859L13.6322 3.65529C13.4503 3.47336 13.2085 3.37323 12.9513 3.37323C12.694 3.37323 12.4522 3.47336 12.2703 3.65529C11.9251 4.0005 11.8976 4.54469 12.1872 4.92196L11.9604 5.13448C11.1502 4.48702 10.1796 4.03234 9.11842 3.84118V3.35659C9.88137 3.328 10.4933 2.69877 10.4933 1.92898C10.4933 1.14103 9.85229 0.5 9.06435 0.5H6.90449C6.11655 0.5 5.47552 1.14103 5.47552 1.92898C5.47552 2.69877 6.08747 3.328 6.85042 3.35659V3.84118C5.77996 4.03404 4.8017 4.49504 3.98725 5.15154L3.79205 4.94874C4.10406 4.57086 4.08345 4.00878 3.73003 3.65529C3.54814 3.47336 3.30628 3.37323 3.04907 3.37323C2.79185 3.37323 2.55003 3.47336 2.36814 3.65529L1.41487 4.60859C1.23298 4.79048 1.13281 5.03227 1.13281 5.28952C1.13281 5.54677 1.23298 5.78856 1.41487 5.97045C1.59676 6.15238 1.83862 6.25251 2.09584 6.25251C2.31346 6.25251 2.51997 6.18055 2.68858 6.04831L2.90129 6.26934C2.08797 7.34025 1.60451 8.67472 1.60451 10.1201C1.60451 13.638 4.46655 16.5 7.98444 16.5C11.5023 16.5 14.3643 13.638 14.3643 10.1201C14.3643 8.66466 13.8741 7.32172 13.0505 6.24703L13.2853 6.02691C13.4586 6.17287 13.6754 6.25251 13.9045 6.25251ZM13.2299 10.1201C13.2299 13.0124 10.8768 15.3656 7.98444 15.3656C5.09208 15.3656 2.73895 13.0124 2.73895 10.1201C2.73895 7.22775 5.09208 4.87462 7.98444 4.87462C10.8768 4.87466 13.2299 7.22775 13.2299 10.1201Z"
                      fill="#191919"
                    />
                    <path
                      d="M7.98418 6.07544C5.75391 6.07544 3.93945 7.8899 3.93945 10.1202C3.93945 12.3504 5.75391 14.1649 7.98418 14.1649C10.2144 14.1649 12.0289 12.3504 12.0289 10.1202C12.0289 7.8899 10.2144 6.07544 7.98418 6.07544ZM10.1333 8.99204L8.79761 10.1618C8.78774 10.3564 8.70939 10.5482 8.56078 10.6968C8.24234 11.0152 7.72602 11.0152 7.40758 10.6968C7.08914 10.3783 7.08914 9.862 7.40758 9.54357C7.58376 9.36739 7.82044 9.28911 8.05073 9.30791L9.38585 8.13864C9.62155 7.93229 9.97984 7.95596 10.1863 8.19162C10.3927 8.42732 10.3689 8.78565 10.1333 8.99204Z"
                      fill="#191919"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_511_22270"
                    >
                      <rect
                        fill="white"
                        height="16"
                        transform="translate(0 0.5)"
                        width="16"
                      />
                    </clippath>
                  </defs>
                </svg>
                <p
                  class="shops__contacts__text"
                >
                  Schedule: Mon-Sun: 
                  11:00 – 20:00
                </p>
              </div>
            </div>
          </div>
          <div
            class="shops__content"
          >
            <h3
              class="shops__city"
            >
              dnipro
            </h3>
            <div
              class="shops__contacts"
            >
              <div
                class="shops__contacts__item"
              >
                <svg
                  fill="none"
                  height="12"
                  viewBox="0 0 12 12"
                  width="12"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M5.99922 12C7.05876 12 10.7992 7.244 10.7992 4.66667C10.7992 2.08934 8.65019 0 5.99922 0C3.34825 0 1.19922 2.08934 1.19922 4.66667C1.19922 7.244 4.93968 12 5.99922 12ZM5.99922 6.66667C7.13535 6.66667 8.05635 5.77124 8.05635 4.66667C8.05635 3.5621 7.13535 2.66667 5.99922 2.66667C4.86309 2.66667 3.94209 3.5621 3.94209 4.66667C3.94209 5.77124 4.86309 6.66667 5.99922 6.66667Z"
                    fill="#191919"
                    fill-rule="evenodd"
                  />
                </svg>
                <p
                  class="shops__contacts__text"
                >
                  Address: 
                  Dnipro, ave. D.Yavornytskogo, 50
                </p>
              </div>
              <div
                class="shops__contacts__item"
              >
                <svg
                  fill="none"
                  height="17"
                  viewBox="0 0 16 17"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M1.33791 3.52769C1.27065 2.57483 2.07752 1.83325 3.03079 1.83325H5.07135C5.56069 1.83325 6.00203 2.12809 6.19023 2.58072L7.04351 4.63289C7.25138 5.1328 7.10385 5.71021 6.68185 6.04853L6.14994 6.47495C5.91257 6.66524 5.83187 6.99338 5.97261 7.26307C6.69591 8.64934 7.76491 9.82581 9.06491 10.6777C9.33931 10.8575 9.70178 10.786 9.90651 10.5295L10.4608 9.83527C10.7984 9.41241 11.3746 9.26454 11.8734 9.47287L13.9214 10.3279C14.3731 10.5165 14.6673 10.9588 14.6673 11.4491V12.1577H14.6399V13.4665C14.6399 14.4217 13.8999 15.2303 12.949 15.1626C6.86871 14.7301 1.76896 9.63441 1.33791 3.52769Z"
                    fill="#191919"
                  />
                </svg>
                <a
                  class="shops__contacts__text"
                  href="tel:+380505000096"
                >
                  Phone: +380(50) 500 00 96
                </a>
              </div>
              <div
                class="shops__contacts__item"
              >
                <svg
                  fill="none"
                  height="17"
                  viewBox="0 0 16 17"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    clip-path="url(#clip0_511_22270)"
                  >
                    <path
                      d="M13.9045 6.25251C14.1618 6.25251 14.4036 6.15238 14.5855 5.97049C14.7674 5.78856 14.8676 5.54677 14.8676 5.28952C14.8676 5.03227 14.7674 4.79048 14.5855 4.60859L13.6322 3.65529C13.4503 3.47336 13.2085 3.37323 12.9513 3.37323C12.694 3.37323 12.4522 3.47336 12.2703 3.65529C11.9251 4.0005 11.8976 4.54469 12.1872 4.92196L11.9604 5.13448C11.1502 4.48702 10.1796 4.03234 9.11842 3.84118V3.35659C9.88137 3.328 10.4933 2.69877 10.4933 1.92898C10.4933 1.14103 9.85229 0.5 9.06435 0.5H6.90449C6.11655 0.5 5.47552 1.14103 5.47552 1.92898C5.47552 2.69877 6.08747 3.328 6.85042 3.35659V3.84118C5.77996 4.03404 4.8017 4.49504 3.98725 5.15154L3.79205 4.94874C4.10406 4.57086 4.08345 4.00878 3.73003 3.65529C3.54814 3.47336 3.30628 3.37323 3.04907 3.37323C2.79185 3.37323 2.55003 3.47336 2.36814 3.65529L1.41487 4.60859C1.23298 4.79048 1.13281 5.03227 1.13281 5.28952C1.13281 5.54677 1.23298 5.78856 1.41487 5.97045C1.59676 6.15238 1.83862 6.25251 2.09584 6.25251C2.31346 6.25251 2.51997 6.18055 2.68858 6.04831L2.90129 6.26934C2.08797 7.34025 1.60451 8.67472 1.60451 10.1201C1.60451 13.638 4.46655 16.5 7.98444 16.5C11.5023 16.5 14.3643 13.638 14.3643 10.1201C14.3643 8.66466 13.8741 7.32172 13.0505 6.24703L13.2853 6.02691C13.4586 6.17287 13.6754 6.25251 13.9045 6.25251ZM13.2299 10.1201C13.2299 13.0124 10.8768 15.3656 7.98444 15.3656C5.09208 15.3656 2.73895 13.0124 2.73895 10.1201C2.73895 7.22775 5.09208 4.87462 7.98444 4.87462C10.8768 4.87466 13.2299 7.22775 13.2299 10.1201Z"
                      fill="#191919"
                    />
                    <path
                      d="M7.98418 6.07544C5.75391 6.07544 3.93945 7.8899 3.93945 10.1202C3.93945 12.3504 5.75391 14.1649 7.98418 14.1649C10.2144 14.1649 12.0289 12.3504 12.0289 10.1202C12.0289 7.8899 10.2144 6.07544 7.98418 6.07544ZM10.1333 8.99204L8.79761 10.1618C8.78774 10.3564 8.70939 10.5482 8.56078 10.6968C8.24234 11.0152 7.72602 11.0152 7.40758 10.6968C7.08914 10.3783 7.08914 9.862 7.40758 9.54357C7.58376 9.36739 7.82044 9.28911 8.05073 9.30791L9.38585 8.13864C9.62155 7.93229 9.97984 7.95596 10.1863 8.19162C10.3927 8.42732 10.3689 8.78565 10.1333 8.99204Z"
                      fill="#191919"
                    />
                  </g>
                  <defs>
                    <clippath
                      id="clip0_511_22270"
                    >
                      <rect
                        fill="white"
                        height="16"
                        transform="translate(0 0.5)"
                        width="16"
                      />
                    </clippath>
                  </defs>
                </svg>
                <p
                  class="shops__contacts__text"
                >
                  Schedule: Mon-Sun: 
                  10:00 – 21:00
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="shops container"
    >
      <div
        class="shops__map"
      />
      <div
        class="shops__content__block"
      >
        <div
          class="shops__content"
        >
          <h3
            class="shops__city"
          >
            dnipro
          </h3>
          <div
            class="shops__contacts"
          >
            <div
              class="shops__contacts__item"
            >
              <svg
                fill="none"
                height="12"
                viewBox="0 0 12 12"
                width="12"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M5.99922 12C7.05876 12 10.7992 7.244 10.7992 4.66667C10.7992 2.08934 8.65019 0 5.99922 0C3.34825 0 1.19922 2.08934 1.19922 4.66667C1.19922 7.244 4.93968 12 5.99922 12ZM5.99922 6.66667C7.13535 6.66667 8.05635 5.77124 8.05635 4.66667C8.05635 3.5621 7.13535 2.66667 5.99922 2.66667C4.86309 2.66667 3.94209 3.5621 3.94209 4.66667C3.94209 5.77124 4.86309 6.66667 5.99922 6.66667Z"
                  fill="#191919"
                  fill-rule="evenodd"
                />
              </svg>
              <p
                class="shops__contacts__text"
              >
                Address: 
                Dnipro, blv. Zorianyi, 1a
              </p>
            </div>
            <div
              class="shops__contacts__item"
            >
              <svg
                fill="none"
                height="17"
                viewBox="0 0 16 17"
                width="16"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M1.33791 3.52769C1.27065 2.57483 2.07752 1.83325 3.03079 1.83325H5.07135C5.56069 1.83325 6.00203 2.12809 6.19023 2.58072L7.04351 4.63289C7.25138 5.1328 7.10385 5.71021 6.68185 6.04853L6.14994 6.47495C5.91257 6.66524 5.83187 6.99338 5.97261 7.26307C6.69591 8.64934 7.76491 9.82581 9.06491 10.6777C9.33931 10.8575 9.70178 10.786 9.90651 10.5295L10.4608 9.83527C10.7984 9.41241 11.3746 9.26454 11.8734 9.47287L13.9214 10.3279C14.3731 10.5165 14.6673 10.9588 14.6673 11.4491V12.1577H14.6399V13.4665C14.6399 14.4217 13.8999 15.2303 12.949 15.1626C6.86871 14.7301 1.76896 9.63441 1.33791 3.52769Z"
                  fill="#191919"
                />
              </svg>
              <a
                class="shops__contacts__text"
                href="tel:+380505000096"
              >
                Phone: +380(50) 500 00 96
              </a>
            </div>
            <div
              class="shops__contacts__item"
            >
              <svg
                fill="none"
                height="17"
                viewBox="0 0 16 17"
                width="16"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_511_22270)"
                >
                  <path
                    d="M13.9045 6.25251C14.1618 6.25251 14.4036 6.15238 14.5855 5.97049C14.7674 5.78856 14.8676 5.54677 14.8676 5.28952C14.8676 5.03227 14.7674 4.79048 14.5855 4.60859L13.6322 3.65529C13.4503 3.47336 13.2085 3.37323 12.9513 3.37323C12.694 3.37323 12.4522 3.47336 12.2703 3.65529C11.9251 4.0005 11.8976 4.54469 12.1872 4.92196L11.9604 5.13448C11.1502 4.48702 10.1796 4.03234 9.11842 3.84118V3.35659C9.88137 3.328 10.4933 2.69877 10.4933 1.92898C10.4933 1.14103 9.85229 0.5 9.06435 0.5H6.90449C6.11655 0.5 5.47552 1.14103 5.47552 1.92898C5.47552 2.69877 6.08747 3.328 6.85042 3.35659V3.84118C5.77996 4.03404 4.8017 4.49504 3.98725 5.15154L3.79205 4.94874C4.10406 4.57086 4.08345 4.00878 3.73003 3.65529C3.54814 3.47336 3.30628 3.37323 3.04907 3.37323C2.79185 3.37323 2.55003 3.47336 2.36814 3.65529L1.41487 4.60859C1.23298 4.79048 1.13281 5.03227 1.13281 5.28952C1.13281 5.54677 1.23298 5.78856 1.41487 5.97045C1.59676 6.15238 1.83862 6.25251 2.09584 6.25251C2.31346 6.25251 2.51997 6.18055 2.68858 6.04831L2.90129 6.26934C2.08797 7.34025 1.60451 8.67472 1.60451 10.1201C1.60451 13.638 4.46655 16.5 7.98444 16.5C11.5023 16.5 14.3643 13.638 14.3643 10.1201C14.3643 8.66466 13.8741 7.32172 13.0505 6.24703L13.2853 6.02691C13.4586 6.17287 13.6754 6.25251 13.9045 6.25251ZM13.2299 10.1201C13.2299 13.0124 10.8768 15.3656 7.98444 15.3656C5.09208 15.3656 2.73895 13.0124 2.73895 10.1201C2.73895 7.22775 5.09208 4.87462 7.98444 4.87462C10.8768 4.87466 13.2299 7.22775 13.2299 10.1201Z"
                    fill="#191919"
                  />
                  <path
                    d="M7.98418 6.07544C5.75391 6.07544 3.93945 7.8899 3.93945 10.1202C3.93945 12.3504 5.75391 14.1649 7.98418 14.1649C10.2144 14.1649 12.0289 12.3504 12.0289 10.1202C12.0289 7.8899 10.2144 6.07544 7.98418 6.07544ZM10.1333 8.99204L8.79761 10.1618C8.78774 10.3564 8.70939 10.5482 8.56078 10.6968C8.24234 11.0152 7.72602 11.0152 7.40758 10.6968C7.08914 10.3783 7.08914 9.862 7.40758 9.54357C7.58376 9.36739 7.82044 9.28911 8.05073 9.30791L9.38585 8.13864C9.62155 7.93229 9.97984 7.95596 10.1863 8.19162C10.3927 8.42732 10.3689 8.78565 10.1333 8.99204Z"
                    fill="#191919"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_511_22270"
                  >
                    <rect
                      fill="white"
                      height="16"
                      transform="translate(0 0.5)"
                      width="16"
                    />
                  </clippath>
                </defs>
              </svg>
              <p
                class="shops__contacts__text"
              >
                Schedule: Mon-Sun: 
                11:00 – 20:00
              </p>
            </div>
          </div>
        </div>
        <div
          class="shops__content"
        >
          <h3
            class="shops__city"
          >
            dnipro
          </h3>
          <div
            class="shops__contacts"
          >
            <div
              class="shops__contacts__item"
            >
              <svg
                fill="none"
                height="12"
                viewBox="0 0 12 12"
                width="12"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M5.99922 12C7.05876 12 10.7992 7.244 10.7992 4.66667C10.7992 2.08934 8.65019 0 5.99922 0C3.34825 0 1.19922 2.08934 1.19922 4.66667C1.19922 7.244 4.93968 12 5.99922 12ZM5.99922 6.66667C7.13535 6.66667 8.05635 5.77124 8.05635 4.66667C8.05635 3.5621 7.13535 2.66667 5.99922 2.66667C4.86309 2.66667 3.94209 3.5621 3.94209 4.66667C3.94209 5.77124 4.86309 6.66667 5.99922 6.66667Z"
                  fill="#191919"
                  fill-rule="evenodd"
                />
              </svg>
              <p
                class="shops__contacts__text"
              >
                Address: 
                Dnipro, ave. D.Yavornytskogo, 50
              </p>
            </div>
            <div
              class="shops__contacts__item"
            >
              <svg
                fill="none"
                height="17"
                viewBox="0 0 16 17"
                width="16"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M1.33791 3.52769C1.27065 2.57483 2.07752 1.83325 3.03079 1.83325H5.07135C5.56069 1.83325 6.00203 2.12809 6.19023 2.58072L7.04351 4.63289C7.25138 5.1328 7.10385 5.71021 6.68185 6.04853L6.14994 6.47495C5.91257 6.66524 5.83187 6.99338 5.97261 7.26307C6.69591 8.64934 7.76491 9.82581 9.06491 10.6777C9.33931 10.8575 9.70178 10.786 9.90651 10.5295L10.4608 9.83527C10.7984 9.41241 11.3746 9.26454 11.8734 9.47287L13.9214 10.3279C14.3731 10.5165 14.6673 10.9588 14.6673 11.4491V12.1577H14.6399V13.4665C14.6399 14.4217 13.8999 15.2303 12.949 15.1626C6.86871 14.7301 1.76896 9.63441 1.33791 3.52769Z"
                  fill="#191919"
                />
              </svg>
              <a
                class="shops__contacts__text"
                href="tel:+380505000096"
              >
                Phone: +380(50) 500 00 96
              </a>
            </div>
            <div
              class="shops__contacts__item"
            >
              <svg
                fill="none"
                height="17"
                viewBox="0 0 16 17"
                width="16"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  clip-path="url(#clip0_511_22270)"
                >
                  <path
                    d="M13.9045 6.25251C14.1618 6.25251 14.4036 6.15238 14.5855 5.97049C14.7674 5.78856 14.8676 5.54677 14.8676 5.28952C14.8676 5.03227 14.7674 4.79048 14.5855 4.60859L13.6322 3.65529C13.4503 3.47336 13.2085 3.37323 12.9513 3.37323C12.694 3.37323 12.4522 3.47336 12.2703 3.65529C11.9251 4.0005 11.8976 4.54469 12.1872 4.92196L11.9604 5.13448C11.1502 4.48702 10.1796 4.03234 9.11842 3.84118V3.35659C9.88137 3.328 10.4933 2.69877 10.4933 1.92898C10.4933 1.14103 9.85229 0.5 9.06435 0.5H6.90449C6.11655 0.5 5.47552 1.14103 5.47552 1.92898C5.47552 2.69877 6.08747 3.328 6.85042 3.35659V3.84118C5.77996 4.03404 4.8017 4.49504 3.98725 5.15154L3.79205 4.94874C4.10406 4.57086 4.08345 4.00878 3.73003 3.65529C3.54814 3.47336 3.30628 3.37323 3.04907 3.37323C2.79185 3.37323 2.55003 3.47336 2.36814 3.65529L1.41487 4.60859C1.23298 4.79048 1.13281 5.03227 1.13281 5.28952C1.13281 5.54677 1.23298 5.78856 1.41487 5.97045C1.59676 6.15238 1.83862 6.25251 2.09584 6.25251C2.31346 6.25251 2.51997 6.18055 2.68858 6.04831L2.90129 6.26934C2.08797 7.34025 1.60451 8.67472 1.60451 10.1201C1.60451 13.638 4.46655 16.5 7.98444 16.5C11.5023 16.5 14.3643 13.638 14.3643 10.1201C14.3643 8.66466 13.8741 7.32172 13.0505 6.24703L13.2853 6.02691C13.4586 6.17287 13.6754 6.25251 13.9045 6.25251ZM13.2299 10.1201C13.2299 13.0124 10.8768 15.3656 7.98444 15.3656C5.09208 15.3656 2.73895 13.0124 2.73895 10.1201C2.73895 7.22775 5.09208 4.87462 7.98444 4.87462C10.8768 4.87466 13.2299 7.22775 13.2299 10.1201Z"
                    fill="#191919"
                  />
                  <path
                    d="M7.98418 6.07544C5.75391 6.07544 3.93945 7.8899 3.93945 10.1202C3.93945 12.3504 5.75391 14.1649 7.98418 14.1649C10.2144 14.1649 12.0289 12.3504 12.0289 10.1202C12.0289 7.8899 10.2144 6.07544 7.98418 6.07544ZM10.1333 8.99204L8.79761 10.1618C8.78774 10.3564 8.70939 10.5482 8.56078 10.6968C8.24234 11.0152 7.72602 11.0152 7.40758 10.6968C7.08914 10.3783 7.08914 9.862 7.40758 9.54357C7.58376 9.36739 7.82044 9.28911 8.05073 9.30791L9.38585 8.13864C9.62155 7.93229 9.97984 7.95596 10.1863 8.19162C10.3927 8.42732 10.3689 8.78565 10.1333 8.99204Z"
                    fill="#191919"
                  />
                </g>
                <defs>
                  <clippath
                    id="clip0_511_22270"
                  >
                    <rect
                      fill="white"
                      height="16"
                      transform="translate(0 0.5)"
                      width="16"
                    />
                  </clippath>
                </defs>
              </svg>
              <p
                class="shops__contacts__text"
              >
                Schedule: Mon-Sun: 
                10:00 – 21:00
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
